vnext/ReactUWP/Views/KeyboardEventHandler.cpp

```diff
@@ -16,6 +16,7 @@ static constexpr auto SHIFT_KEY = "shiftKey";
 static constexpr auto EVENT_PHASE = "handledEventPhase";
 static constexpr auto KEY = "key";
```

Comment:
Is KEY now unreferenced and can be deleted?

Reply:
> Is KEY now unreferenced and can be deleted?

oh I see its still referenced in ToEventData

Reply:
yes. I still use it to raise event.